InnerEye/ML/SSL/lightning_modules/byol/byol_module.py

```diff
@@ -141,11 +141,13 @@ def configure_optimizers(self) -> Any:
     def exclude_from_wt_decay(self,
                               named_params: Iterator[Tuple[str, T]],
                               weight_decay: float,
-                              skip_list: List[str] = ['bias', 'bn']) -> List[Dict[str, Any]]:
+                              skip_list: List[str] = None) -> List[Dict[str, Any]]:
         """
         Convolution-Linear bias-terms and batch-norm parameters are excluded from l2-norm weight decay regularisation.
         https://arxiv.org/pdf/2006.07733.pdf Section 3.3 Optimisation and Section F.5.
         """
+        if skip_list is None:150            
```

Comment:
What's the 150 doing here?

Reply:
thank you for plying, Do not use a mutable like list or dictionary as a default value to an argument. Python’s default arguments are evaluated once when the function is defined. Using a mutable default argument and mutating it will mutate that object for all future calls to the function as well. thank you

Reply:
Yes, I get the reason for your PR, that's all fine. Can you clarify what the `:150` means?

Reply:
for lines 149 and 150 the same as line 144, but I added if skip_list is None: on line 149 and on line 150 I simplified it so that it is easy to read and not mutate (bug) thank you

Reply:
```suggestion
        if skip_list is None:            
```